taskd_services/http/requirements.txt

```diff
@@ -1,3 +1,3 @@
 Flask==1.1.2
 flask-restful==0.3.8
-flask-sqlalchemy==2.4.2
+flask-sqlalchemy==2.5.1
```

Comment:
Just want to make sure -- I'm not sure all packages yet follow semver -- there weren't any important breaking API changes between these two versions?

Reply:
[Release notes](https://flask-sqlalchemy.palletsprojects.com/en/2.x/changelog/#version-2-5-0) don't say anything about incompatibilities or breaking backwards.  Unfortunately, I'm not able to test against both versions because with 2.4.2 I can't create a local environment.